build-support/bin/release-changelog-helper.sh

```diff
@@ -41,8 +41,12 @@ echo "Potentially relevant headers:"
 echo "----------------------------------------------------------------------------------------------------"
 cat <<EOF
 
-API Changes
-~~~~~~~~~~~
+User API Changes
```

Comment:
Very open to better names for this. I considered "End-user API changes" also.

Reply:
IIUC this is accurate, but also alot of words: `Goal, Option and BUILD syntax changes.`